src/bindings/gsettings/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required (VERSION 2.8)
+project (elektrasettings)
+
+find_package (PkgConfig REQUIRED)
+pkg_check_modules (GLIB glib-2.0>=2.48.0 QUIET)
+pkg_check_modules (GMODULE gmodule-2.0>=2.48 QUIET)
+pkg_check_modules (GIO gio-2.0>=2.48 QUIET)
+if (!GLIB_FOUND)
+remove_binding (glib "glib needed for gsettings backend")
+return ()
+endif ()
+if (!GMODULE_FOUND)
+remove_binding (glib "gmodule needed for gsettings backend")
+return ()
+endif ()
+if (!GIO_FOUND)
+remove_binding (glib "gio needed for gsettings backend")
+return ()
+endif ()
+
+include_directories (${GLIB_INCLUDE_DIRS})
+add_library (elektrasettings SHARED elektrasettingsbackend.c ${ELEKTRA_HEADERS})
+add_dependencies (elektrasettings elektra_config_headers)
+set (GSETTINGS_MODULE_PRIORITY 200 CACHE STRING "GIO Module Priority. Lower then 100 means dconf backend will be default.")
```

Comment:
(not to be fixed now, but maybe add as todo: can the module priority be defined via an Elektra key?)